### PR TITLE
update no data for monitor slos

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by roxanne-moslehi** [comment]
@zhengshizhao can you help review this? updating our docs for no data behavior for monitors, but I want to confirm it's correct, and I don't know what to put for "evaluate as zero". In my past notes, I have "SLO will match monitor (OK = uptime, alert = downtime)" for this case?

File: content/en/service_management/service_level_objectives/monitor.md [hunk 1]
```diff
@@ -112,7 +112,19 @@ Muting a monitor does not affect the SLO calculation.
 To exclude time periods from an SLO calculation, use the [SLO status corrections][5] feature.
 
 ### Missing data
-When you create a metric monitor or service check, you choose whether it sends an alert when data is missing. This configuration affects how a monitor-based SLO calculation interprets missing data. For monitors configured to ignore missing data, time periods with missing data are treated as OK (uptime) by the SLO. For monitors configured to alert on missing data, time periods with missing data are treated as ALERT (downtime) by the SLO.
+#### Metric monitors
+When you create a metric monitor, you choose [how the monitor will handle missing data][6]. This configuration affects how a monitor-based SLO calculation interprets missing data:
+
+| Monitor configuration     | SLO calculation of missing data |
+|---------------------------|---------------------------------|
+| `Evaluate as zero`        | ???                             |
```
Comment (zhengshizhao) [inline content/en/service_management/service_level_objectives/monitor.md:120]: Up to underlying monitor status set by comparing zero with alert threshold.  For example, if the alert threshold is set to > 10,  the monitor status would be set to OK. Thus,  Uptime in SLO calculation

**Comment by roxanne-moslehi** [comment]
Hi @estherk15! This is ready to merge. FYI @zhengshizhao

**Review by zhengshizhao — APPROVED** [review]
thanks for the update!

**Review by aliciascott — APPROVED** [review]

**Pull request merged**